packages/uui-menu-item/lib/uui-menu-item.element.ts

```diff
@@ -553,7 +554,7 @@ export class UUIMenuItemElement extends SelectOnlyMixin(
 
       #icon {
         display: inline-flex;
-        font-size: 16px;
+        font-size: 15.66px;
```

Comment:
Interesting change, @skttl could you put a few words to this number of pixels.

From a visual stand-point icons would have the same size across the application. So having this little difference seems odd. — I'm open for changing the icon-size in general, but is has to be generally better then :-D

Reply:
Yes, it is a weird pixel size.

`uui-icon` has a size of 1.15em*1.15em. With 16px that makes 18.39 pixels, which made the browser antialias the icon in a less optimal way.

I could have bumped `uui-icon` to 1.125em*1.125em as @bjarnef mentions, but that would change icon sizes across the whole backoffice.

Instead going for a font size of 15.66, the icon in the navigation would end up as 18x18 pixels, and have a lot better antialiasing.

Reply:
Okay, thanks for explaining. :-)

So with that perspective I think @bjarnef 's suggestion is the one to go with, cause then we have aligned the size across the application.

But that is from a technical idea that 18px is better than 18.39px, but the reality is that the lucide icons are made for 24px, so the coordinates of the SVG will be off in both cases. So I went to have a look at the actual rendering, in a few cases and there was it seems to me that it in general got worse.

Here is one of them, where the top one is the old and the bottom the new(from a icon perspective, did not include the caret change)
![image](https://github.com/user-attachments/assets/eab1bcb0-d0f3-425b-be57-dcedd194b720)

And in this specific case the top one seems less smudged(anti-aliases badly) and the bottom one quite smudged.

So from that perspective, 'how it actually renders', I would say its better to keep it as it is, the 1.15.

What are you thoughts on that? :-) 

BTW. I love the other changes, and I love this kind of attention to the details. We should make the UI as sharp as possible, specially for the Backoffice where these details are the UI. :-D

Reply:
I think the weird pixel size may just cause more issues in future, when it should ensure the base icon size is correct depending on the icon library.

Furthermore an overview like this may also help.
https://github.com/umbraco/Umbraco.UI/discussions/887

Currently it seems we only have an overview of essential icons.

Reply:
@nielslyngsoe I do agree, the upper one looks sharper to me too, maybe there is a difference in browser rendering? Which browser are you using?

When doing the PR, I compared to the icon sizes in the old backoffice, and they were 20px. I guess 24 is going to be too big though.

Reply:
@skttl yes, that is the problem, 24px is too big for our use case. :-D

But I'm happy with adapting the icons in general, so let me know once that is in place then we can have this merged :-)

Reply:
Hi all, 

I have applied the suggestion from @bjarnef. Let's get this merged. Thanks!